### ISSUE TITLE
Installation Python 3.9 fails

### ISSUE THREAD

**Issue by charleygros** [ivadomed/ivadomed#556]
Installation with Python 3.9 does not work
```
pip install ivadomed
Collecting ivadomed
  Downloading ivadomed-2.6.0-py3-none-any.whl (139 kB)
     |████████████████████████████████| 139 kB 8.9 MB/s
  Downloading ivadomed-2.5.0-py3-none-any.whl (137 kB)
     |████████████████████████████████| 137 kB 17.9 MB/s
  Downloading ivadomed-2.4.0-py3-none-any.whl (132 kB)
     |████████████████████████████████| 132 kB 23.7 MB/s
  Downloading ivadomed-2.3.1-py3-none-any.whl (129 kB)
     |████████████████████████████████| 129 kB 24.1 MB/s
  Downloading ivadomed-2.2.1-py3-none-any.whl (127 kB)
     |████████████████████████████████| 127 kB 28.2 MB/s
  Downloading ivadomed-2.2.0-py3-none-any.whl (127 kB)
     |████████████████████████████████| 127 kB 23.9 MB/s
  Downloading ivadomed-2.1.0-py3-none-any.whl (110 kB)
     |████████████████████████████████| 110 kB 30.8 MB/s
  Downloading ivadomed-2.0.2-py3-none-any.whl (108 kB)
     |████████████████████████████████| 108 kB 29.9 MB/s
  Downloading ivadomed-2.0.1-py3-none-any.whl (108 kB)
     |████████████████████████████████| 108 kB 30.9 MB/s
  Downloading ivadomed-1.2.1-py3-none-any.whl (68 kB)
     |████████████████████████████████| 68 kB 1.8 MB/s
  Downloading ivadomed-1.2-py3-none-any.whl (68 kB)
     |████████████████████████████████| 68 kB 5.0 MB/s
  Downloading ivadomed-1.1-py3-none-any.whl (68 kB)
     |████████████████████████████████| 68 kB 4.6 MB/s
  Downloading ivadomed-1.0-py3-none-any.whl (68 kB)
     |████████████████████████████████| 68 kB 2.4 MB/s
  Downloading ivadomed-0.14-py3-none-any.whl (68 kB)
     |████████████████████████████████| 68 kB 4.8 MB/s
  Downloading ivadomed-0.13-py3-none-any.whl (68 kB)
     |████████████████████████████████| 68 kB 457 kB/s
  Downloading ivadomed-0.12-py3-none-any.whl (66 kB)
     |████████████████████████████████| 66 kB 3.5 MB/s
  Downloading ivadomed-0.11-py3-none-any.whl (66 kB)
     |████████████████████████████████| 66 kB 3.3 MB/s

ERROR: Cannot install ivadomed==0.11, ivadomed==0.12, ivadomed==0.13, ivadomed==0.14, ivadomed==1.0, ivadomed==1.1, ivadomed==1.2, ivadomed==1.2.1, ivadomed==2.0.1, ivadomed==2.0.2, ivadomed==2.1.0, ivadomed==2.2.0, ivadomed==2.2.1, ivadomed==2.3.1, ivadomed==2.4.0, ivadomed==2.5.0 and ivadomed==2.6.0 because these package versions have conflicting dependencies.
```

Avenues:
- loosen the range of package versions specified
- remove package versions to allow pip attempt to solve the dependency conflict
- https://pip.pypa.io/en/latest/user_guide/#fixing-conflicting-dependencies

**Comment by lrouhier** [comment]
I have the same output as in the review. I do think the second part is the most interresting (see below) 
~~~
The conflict is caused by:
    ivadomed 2.6.0 depends on torchvision==0.6.0
    ivadomed 2.5.0 depends on torchvision==0.6.0
    ivadomed 2.4.0 depends on torchvision==0.6.0
    ivadomed 2.3.1 depends on onnxruntime
    ivadomed 2.2.1 depends on onnxruntime
    ivadomed 2.2.0 depends on onnxruntime
    ivadomed 2.1.0 depends on onnxruntime
    ivadomed 2.0.2 depends on onnxruntime
    ivadomed 2.0.1 depends on onnxruntime
    ivadomed 1.2.1 depends on onnxruntime
    ivadomed 1.2 depends on onnxruntime
    ivadomed 1.1 depends on torchvision>=0.6.0
    ivadomed 1.0 depends on torchvision>=0.6.0
    ivadomed 0.14 depends on torchvision>=0.6.0
    ivadomed 0.13 depends on torchvision>=0.6.0
    ivadomed 0.12 depends on torchvision>=0.6.0
    ivadomed 0.11 depends on torchvision>=0.6.0

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict

ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/user_guide/#fixing-conflicting-dependencies
~~~

The problem is the following: torchvision 0.6.0 does not seem to be available in python 3.9, neither is onnxruntime. 
Unfortunately we do depends on these two packages (because torch 1.5.0 depends on torchvision 0.6.0), plus we need onnxruntime to use our onnx model and to package the models. 

For now, I think that we should specify that we do not support python 3.9..

**Comment by charleygros** [comment]
> For now, I think that we should specify that we do not support python 3.9..

Sounds reasonable, thanks @lrouhier !!
